extras/java8-tests/src/test/java/org/apache/spark/streaming/Java8APISuite.java

```diff
@@ -440,8 +441,13 @@ public void testPairFlatMap() {
   public static <T extends Comparable<T>> void assertOrderInvariantEquals(
     List<List<T>> expected, List<List<T>> actual) {
     expected.forEach((List<T> list) -> Collections.sort(list));
-    actual.forEach((List<T> list) -> Collections.sort(list));
-    Assert.assertEquals(expected, actual);
+    ArrayList<ArrayList<T>> sortedActual = new ArrayList<ArrayList<T>>();
+    actual.forEach((List<T> list) -> {
```

Comment:
Really? it's a little less code and matches the functional style of other (Scala) code. Performance is not an issue here. However I forgot the collect(). How about:

```
List<List<T>> actualSorted = actual.stream().
  map(l -> new ArrayList<>(l)).map(Collections::sort).
  collect(Collectors.toList());
```